src/components/Card/Card.tsx

```diff
@@ -0,0 +1,45 @@
+import { FC } from "react";
+import "./Card.css";
+
+type CardProps = {
+    card: number;
+    index: number;
+    selectedCards: number[];
+    cardsMatch: number[];
+    flipCard: (index: number) => void;
+    timeBeforeGame: boolean;
+}
+
+type IsMutchedType = ' is_mutched' | '';
+type IsActiveType = ' active' | '';
+type IsTimeoutType = ' is_timeout' | '';
```

Comment:
This looks a little hacky, so I would suggest a little different approach

You can form an array of these strings and then `.join(' ')` them with a space as a separator